cypress/elements/BasePage.js

```diff
@@ -0,0 +1,33 @@
+import { NEW_ITEMS } from './../helpers/toDoArray'
```

Comment:
Fix [fixture concept](https://docs.cypress.io/api/commands/fixture) for data operations